crates/edr_evm/src/runtime.rs

```diff
@@ -155,6 +155,7 @@ where
     StateErrorT: Debug + Send,
 {
     cfg.disable_balance_check = true;
+    cfg.disable_base_fee = true;
```

Comment:
I had to add this as the block's base fee check was failing. @fvictorio was EthereumJS also disabling the checks for comparing the transaction's max fee against the header's base fee?

Reply:
Not that I remember, and I can't find anything like that in the `ethereums/main` branch of Hardhat. These are the "skip validation" flags we used:

- `skipConsensusFormatValidation` (when creating a block)
- `skipNonce`, `skipBalance`, `skipBlockGasLimitValidation` and `skipHardForkValidation` (when running a call or estimate gas)

Can you give me more context? Where is that check run? In which context (mining a block? making a historical 
 call?) Why would a transaction have a fee below the block's base fee, and why would we want to accept it?